docs/source/examples.rst

```diff
@@ -122,12 +122,13 @@ A complete list of examples is given below:
 .. automodule:: 17b_zeromq_multi_openfast
 .. automodule:: 18_pitch_offsets
 .. automodule:: 19_update_discon_version
-.. automodule:: 20_active_wake_control
+.. .. automodule:: 20_active_wake_control
```

Comment:
Is commenting out Example 20 here intentional?

Reply:
Yes, but I'll revert it until I come up with a better solution.  This example is well documented in the python script, but it's not in the rst format.  I'm slowly working on expanding the example documentation.  I wonder if we can point to another rst file within this automodule set up?

Reply:
The easiest way would be to just add the needed rst text between automodule 19 and 21. Not sure if it would match the formatting but should be OK at this stage.